DuggaSys/diagram.js

```diff
@@ -5242,55 +5294,23 @@ function loadAppearanceForm() {
             appearanceObjects.push(object);
         }
     }
-    if(appearanceObjects.length < 1) return;
-
-    //Stores current index and maximum index for each type of selected object
-    //Current index will be used for comma separation, to only create comma when max is more than current
-    const indexes = {};
-
-    //Reduce the selected objects array to a Map with key symbolKind and value the number of times that symbolKind occurs
-    //Iterate through the map and put the correct values in the indexes object
-    appearanceObjects.reduce((result, object) => {
-        result.set(object.symbolkind || 0, (result.get(object.symbolkind || 0) || 0) + 1);
-        return result;
-    }, new Map()).forEach((value, key) => {
-        if(typeof indexes[key] === "undefined") {
-            indexes[key] = {
-                current: 0,
-                max: 0
-            };
-        }
-        indexes[key].max += value;
-    });
-
-    //Get all unique types of selected objects
-    const types = Object.keys(indexes).map(Number);
```

Comment:
I'm not sure why all of this is shown. I haven't touches these lines... Branch says no conflict so should be fine?

Reply:
This is absolutely not fine.